### PR TITLE
INTGRTN-1723: updated-text-community-integration

### PR DESCRIPTION
**Pull request by Alex-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

**Comment by Alex-MultiSafepay** [comment]
@Qadira-MultiSafepay is it now ok?

File: content/integrations/community/Community integration requirements/_index.md [hunk 1]
```diff
@@ -11,15 +11,14 @@ hideMenu: '.'
 
 At MultiSafepay, our developers code on a daily basis to process in-house transactions. However, there are also many fantastic developers worldwide who code wrappers, plugins, applications and other various code in order to work with us.
 
-We are happy to accompany you starting with a place on our documentation page as well as assist you on marketing level. If you would like for your project to appear on our documentation page, please contact the MultiSafepay Integration Team at <integration@multisafepay.com>
+We are happy to give a place on our documentation page if you have created an integration with us. If you would like for your project to appear on our documentation page, please contact the MultiSafepay Integration Team at integration@multisafepay.com
```
Comment (Atiq-MultiSafepay) [inline content/integrations/community/Community integration requirements/_index.md:14]: We would be delighted to offer a place on our documentation page if you have created an integration with us. If you would like for your project to appear on our documentation page, please contact the MultiSafepay Integration Team at integration@multisafepay.com

File: content/integrations/community/Community integration requirements/_index.md [hunk 2]
```diff
@@ -11,15 +11,14 @@ hideMenu: '.'
...
-* The readme should indicate that full responsibility lies with the developer of the community integration. This includes customer support and maintenance of the code. It should also indicate that the integration was __not created by MultiSafepay__
-* If MultiSafepay decides to add the community integration as one of the supported integrations by MultiSafepay, we will have the authority to also remove the community integration without the breach of contract
-* It is required for the community integration to have updates. A support tab would be ideal to have, but is not compulsory.
-* If you have any further questions about being part of the community Integration Team, feel free to contact <integration@multisafepay.com>
+* The [integration information](https://docs.multisafepay.com/api/#plugin-information) should be visible and provided on the page and further include a detailed description of the configuration of the integration and the usage
+* The responsibility lies with the developer of the integration, and we expect you to provide customer support and maintenance of the code.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/community/Community integration requirements/_index.md:19]: * The responsibility lies with the developer of the integration and we expect you to provide customer support and maintenance of the code.

File: content/integrations/community/Community integration requirements/_index.md [hunk 3]
```diff
@@ -11,15 +11,14 @@ hideMenu: '.'
...
+* On your page it should indicate that the integration was not created by MultiSafepay
```
Comment (Atiq-MultiSafepay) [inline content/integrations/community/Community integration requirements/_index.md:20]: * On the page it should indicate that the integration was not created by MultiSafepay

File: content/integrations/community/Community integration requirements/_index.md [hunk 4]
```diff
@@ -11,15 +11,14 @@ hideMenu: '.'
...
+* If MultiSafepay decides to add the integration as one of the supported integrations by MultiSafepay, we will have the authority to also remove the community integration without the breach of contract
```
Comment (Atiq-MultiSafepay) [inline content/integrations/community/Community integration requirements/_index.md:21]: * If MultiSafepay decides to add the integration as one of the supported integrations, we will have the authority to also remove the community integration without the breach of contract

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/integrations/community/Community integration requirements/_index.md (hunk 1 above)
Comment (Atiq-MultiSafepay) [reply content/integrations/community/Community integration requirements/_index.md:14]: If you would like to share your project with us and appear on DOCS, feel free to contact the MultiSafepay Integration team at integration@multisafepay.om

**Review by Atiq-MultiSafepay — APPROVED** [review]

**Pull request merged**